c2corg_common/attributes.py

```diff
@@ -667,3 +668,85 @@
     'aran',
     'catalunya'
 ]
+
+event_types = [
+    'avalanche',
+    'stone_fall',
+    'falling_ice',
+    'person_fall',
+    'crevasse_fall',
+    'roped_fall',
+    'physical_failure',
+    'lightning',
+    'other'
+]
+
+author_statuses = [
+    'primary_impacted',
+    'secondary_impacted',
+    'internal_witness',
+    'external_witness'
+]
+
+activity_rates = [
+    'activity_rate_150',
```

Comment:
We could take the V5 EN values.
But the purpose is to be able to add in the future some values between 2 values, and to have the possibility to order a list on this field.
In the V6, as there is no more integer value to code the value in the BD, if we use these string IDs to order, we must use strings using a number in prefix or suffix, to be able to order using alphabetical order.